File: app/src/main/java/com/tomclaw/drawa/draw/PaletteImageView.java
package com.tomclaw.drawa.draw;

import android.content.Context;
import android.support.v7.widget.AppCompatImageView;
import android.util.AttributeSet;

/**
 * Created by Solkin on 25.12.2014.
 */
public class PaletteImageView extends AppCompatImageView {
    public PaletteImageView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public PaletteImageView(Context context) {
        super(context);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);
    }
}

File: app/src/main/java/com/tomclaw/drawa/draw/PaletteImageView.kt
package com.tomclaw.drawa.draw

import android.content.Context
import android.support.v7.widget.AppCompatImageView
import android.util.AttributeSet

class PaletteImageView : AppCompatImageView {

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    constructor(context: Context) : super(context)

    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }

}
